### PR TITLE
GH-460: Provide smooth path for users to run Control Center Docker with any tutorial

### PR DESCRIPTION
**Pull request by ybyzek**
Addresses https://github.com/confluentinc/kafka-tutorials/issues/460

This is 💯 opt-in, with no impact to users who do not run the provied bash script.  This is intended to maintain integrity of Kafka Tutorials as being totally for the open source community, while reducing friction for those who are interested in C3.

### CODE REVIEW

**Comment by rspurgeon** [comment]
Uncertain of the value for this to the 'users' of the site (if we are calling users people landing on kafka-tutorials.confluent.io). The prototypical user comes to the rendered kafka-tutorials.confluent.io site and not the Github hosted source repo.   Most users won't see the README.  This isn't a comment about the value to kafka-tutorials developers...

**Comment by rspurgeon** [comment]
Another option to consider is a (hefty) one-liner that we could advertise to users of the site with a "tip" in some way.  From within a tutorial directory the below can run C3 on the compose network (assuming we don't give the networks non-default names and the C3 port binding isn't taken):

`docker run --network="${PWD##*/}_default" -d --env CONTROL_CENTER_BOOTSTRAP_SERVERS=broker:9092 --env CONTROL_CENTER_ZOOKEEPER_CONNECT=zookeeper:2181 --env CONTROL_CENTER_CONNECT_CLUSTER=connect:8083 --env CONTROL_CENTER_KSQL_KSQLDB1_URL=http://ksqldb-server:8088 --env CONTROL_CENTER_KSQL_KSQLDB1_ADVERTISED_URL=http://localhost:8088 --env CONTROL_CENTER_SCHEMA_REGISTRY_URL=http://schema-registry:8081 --env CONTROL_CENTER_REPLICATION_FACTOR=1 --env CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS=1 --env CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_PARTITIONS=1 --env CONFLUENT_METRICS_TOPIC_REPLICATION=1 --env PORT=9021 -p 9021:9021 confluentinc/cp-enterprise-control-center:5.5.0`

I'm wondering if tutorials can render an included shared "tip file" or similar ( I don't know the site capabilities ).  We might want to consider any downsides to advertising these on tutorials that don't run connect, ksqlDB or other components (like the 101 tutorials don't).

**Comment by ybyzek** [comment]
@rspurgeon 

1. This PR targets `users` who want to test locally.  They can be those that go directly to GH or also those that first land on https://kafka-tutorials.confluent.io because every tutorial now has a link to the GH repo source code

2. I wouldn't add a tip: feedback was to leave C3 out of the tutorial website

3. Running a standalone docker won't be enough.  If broker image changes to cp-kafka, we still need an overrides file to cp-server.

...

Since we still need the overrides, we still need to debug why `make` isn't picking up the overrides file

**Comment by ybyzek** [comment]
I now understand what the problem is ;) _Everything_ is being called from the tmp directory, which has the docker-compose file but not the overrides file!

**Comment by ybyzek** [comment]
PR ready for review

File: README.md [hunk 1]
```diff
@@ -305,13 +305,19 @@ docker build -t runner . ; docker run -v ${PWD}/harness_runner:/harness_runner/
 
 ### Run a tutorial
 
-1. End-to-end: execute the harness runner for a single tutorial by calling `make`, across all `dev`, `test`, and `prod` stages, to validate it works end-to-end. Identify the tutorial you want and then run `make`. Note that this destroys all the resources and Docker containers it created, so it cleans up after itself.  Format: `(cd _includes/tutorials/<tutorial name>/<type>/code && make)` where type is one of `ksql | kstreams | kafka`. Example:
+1. (optional) If you want to augment or override a tutorial's Docker environment, set the Docker Compose CLI environment variable `COMPOSE_FILE` to include `docker-compose.yml` and the absolute path to a `docker-compose.override.yml` file.  For example, to use Confluent Control Center with any Kafka Tutorial, set `COMPOSE_FILE` to `docker-compose.yml` and the absolute path to [this docker-compose.override.yml](tools/docker-compose.override.yml).
```
Comment (bbejeck) [inline README.md:308]: FYC - emphasize the absolute path part, but this is a minor comment, so feel free to ignore it.

```suggestion
1. (optional) If you want to augment or override a tutorial's Docker environment, set the Docker Compose CLI environment variable `COMPOSE_FILE` to include `docker-compose.yml` and **_the absolute path_** to a `docker-compose.override.yml` file.  For example, to use Confluent Control Center with any Kafka Tutorial, set `COMPOSE_FILE` to `docker-compose.yml` and the absolute path to [this docker-compose.override.yml](tools/docker-compose.override.yml).
```

**Pull request merged**